docs/user-guide/timeline.md

```diff
@@ -1,114 +1,90 @@
 
-* **Step backward** <img src="../img/button/timeline-step-backward-button.jpg" class="ms-docbutton"/> : return to the previous step
+It is possible to show, on the timeline, only the times actually visible on the current map viewport through the **Map Sync Button** <img src="../img/button/timeline-sync-button.jpg" class="ms-docbutton"/>, as follows:
 
-* **Play** <img src="../img/button/timeline-play-button.jpg" class="ms-docbutton"/> : play the animation
+<img src="../img/timeline/timeline-sync-example.jpg" class="ms-docimage"/>
 
-* **Stop** <img src="../img/button/timeline-stop-button.jpg" class="ms-docbutton"/> : stop the animation
+## Animations
 
-* **Step forward** <img src="../img/button/timeline-step-forward-button.jpg" class="ms-docbutton"/> : go ahead to the next step
+The user is allowed to navigate in the timeline of the layer selected through the following buttons: 
 
-* **Settings** <img src="../img/button/timeline-playback-settings-button.jpg" class="ms-docbutton"/> : open the [Animation Settings](#animation-settings) panel
+<img src="../img/timeline/timeline-animation-buttons.jpg" class="ms-docimage"/>
 
-Clicking the **Play** button, by default, the current time cursor reach the initial time of the ***guide layer*** and the animation starts. You will see the layers' configurations changes on the map and the time progress on the histogram. Following the step sequence, the cursor will shift each time to the next step in a certain time interval, the *frame duration*.
+In order to start the animation the user can click on **Play** button <img src="../img/button/timeline-play-button.jpg" class="ms-docbutton"/>. Once the animation is starts, the layer configuration changes on the map and the time progress on the histogram. Following the sequence of the step, the cursor will shift each time to the next step in a certain time interval, the *frame duration*.
 <br>
-The **Stop** button obviously stops the animation and the current time cursor remains in the last position reached.
+Through the **Stop** button <img src="../img/button/timeline-stop-button.jpg" class="ms-docbutton"/> the user can stop the animation and the current time cursor remains in the last position reached.
 <br>
-The **Step backward** or the **Step forward** buttons allow you to statically change the current time, so by clicking on one of them the cursor changes its position (to the previous or the next step) on the histogram and the date/time values of the control cells will be update accordingly.
+The **Step backward** button <img src="../img/button/timeline-step-backward-button.jpg" class="ms-docbutton"/> and the **Step forward** button <img src="../img/button/timeline-step-forward-button.jpg" class="ms-docbutton"/> allow the user to statically change the current time, so by clicking on one of them the cursor changes its position (to the previous or the next step) on the histogram and the date/time values of the control cells will be update accordingly.
```

Comment:
The **Step backward** button <img src="../img/button/timeline-step-backward-button.jpg" class="ms-docbutton"/> and the **Step forward** button <img src="../img/button/timeline-step-forward-button.jpg" class="ms-docbutton"/> allow the user to change the current time. Therefore, by clicking on one of them, the cursor changes its position (to the previous or the next step) on the histogram, the date/time values of the control cells will be update accordingly and the layers in map are updated too.